### ISSUE TITLE
Suggestion: Super Quick Info for the "Parameter Info" box

### ISSUE THREAD

**Issue by IneedHelp** [wmjordan/Codist#245]
Hello, your Super Quick Info is very attractive in both functionality and aesthetics. I was wondering if there's any chance to bring it to the Parameter Info box in the near future. Currently it looks like this
![image](https://user-images.githubusercontent.com/1366970/222323565-01b0daae-0e8a-4aed-81b8-2bae8826c8dc.png)

which is out of tone with how beautiful Super Quick Info looks.

**Comment by fitdev** [comment]
Yes, this is something I have wanted too. By comparison VS's parameter info box looks very dull.

**Comment by wmjordan** [comment]
Is it because you have install another extension which conflicts with Codist?

It should look like the following:
<img width="485" alt="image" src="https://user-images.githubusercontent.com/12182730/222353661-24c4cb34-96c0-4947-af02-7ec309a7f3b2.png">

**Comment by fitdev** [comment]
@wmjordan I think what is meant is not the tooltip when you hover over a parameter, but a tooltip / info box when you are _typing in_ - VS shows you various overloads and provides a short description of the parameter you are entering (i.e. when writing a method call for example).

**Comment by wmjordan** [comment]
Oh, I see.
You want Codist to replace the Parameter Info box with a better implementation.
Hmm, since the Roslyn is open-sourced, maybe I can learn from them and make some improvements. But it takes time.
I can't guarantee you whether, or when this will be implemented.

**Comment by IneedHelp** [comment]
Yeah, i fully understand that an extension like this will be forever work in progress as numerous options and features can be added to it. I appreciate you're taking into consideration the possibility of adding this at a later time.

**Comment by wmjordan** [comment]
I studied the source code of Roslyn and read that the Parameter Info window has something to do with the auto-completion feature. Replacing one thing might requires replacing both of them. That isn't an easy task.

**Comment by fitdev** [comment]
What about replacing the default small tooltip that appears when you select items in Intellisense dropdown - is it similarly difficult to replace as the parameter info window?

**Comment by wmjordan** [comment]
> is it similarly difficult to replace as the parameter info window?

Unfortunately, yes. See #13, which has been there for almost 5 years.
